.circleci/config.yml

```diff
@@ -49,6 +54,40 @@ commands:
           name: fetch all tags
           command: |
             git fetch --all
+  # creates nightly tag from build number of build_linux job
+  create_nightly_version:
+    steps:
+      - run:
+          name: create and export nightly FILECOIN_BINARY_VERSION
+          command: |
+            echo "export FILECOIN_BINARY_VERSION="nightly-${CIRCLE_BUILD_NUM}-$(echo $CIRCLE_SHA1 | cut -c -6)"" >> $BASH_ENV
```

Comment:
why `export FILECOIN_BINARY_VERSION` first and then `echo $FILECOIN_BINARY_VERSION `  ?
wouldn't ` echo  "nightly-${CIRCLE_BUILD_NUM}-$(echo $CIRCLE_SHA1 | cut -c -6)" > release-version-nightly.txt` suffice ?

Reply:
it's so that the job that calls the `create_nightly_version` command will have `FILECOIN_BINARY_VERSION` exported in the bash env for subsequent steps. 

potentially over engineered for now, I can remove it.